### DISCUSSION TITLE
Two battery packs with JKBMS not adding power on VRM

### DISCUSSION THREAD

**Discussion by githubfrance** [Louisvdw/dbus-serialbattery#549] [General]
Hi Louis,
To get 400Ah I added a new battery pack to my previous installation. Both are identical, with JKBMS.
I could see immediately the new pack on Venus OS
 
![VENUS OS](https://user-images.githubusercontent.com/18025874/234224409-74a79296-1b78-478b-ba35-b33b0120454c.png)

But the Amps are not added  on the VRM
![VRM](https://user-images.githubusercontent.com/18025874/234224559-d28543b5-47e2-47cf-a5b3-7c2241f085e0.png)

Is there something I can do or I did wrong ?

thank you 
Patrick

**Comment by mr-manuel** [comment]
Hi, in the overview only the selected battery monitor is shown. If you want to see both you have to aggregate the batteries.

There are two possibilities to do this:

1. https://github.com/Dr-Gigavolt/dbus-aggregate-batteries
2. https://github.com/mr-manuel/venus-os_dbus-mqtt-battery

See also https://github.com/Louisvdw/dbus-serialbattery/issues/8

**Comment by seamaster101** [comment]
@githubfrance you can install and use https://github.com/pulquero/BatteryAggregator it does exactly what you need
